### DISCUSSION TITLE
Load/Unload Configure New Default Extrude Speed

### DISCUSSION THREAD

**Discussion by codejockey1** [KlipperScreen/KlipperScreen#1126] [Q&A]
How can I set the default extrude speed for KlipperScreen? I have the load/unload macros set up and they accept speed as a parameter. The default extrude/retract speed in KlipperScreen is 2mm/sec. My Mainsail defaults to 5mm/sec and I want to set KlipperScreen to match but I don't see anything in the docs where I can set the default for load/unload on the Extrude screen.

**Comment by alfrix** [comment]
it is always the second entry, not configurable
you can change the list and place whatever you want in that place to make it default
https://klipperscreen.readthedocs.io/en/latest/Configuration/#printer-options
